### DISCUSSION TITLE
Welcome to metarank Discussions!

### DISCUSSION THREAD

**Discussion by shuttie** [metarank/metarank#300] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by moferry96** [comment]
Hi,
I'm Mohammadhossein, and trying to get an collaborative recommender system up and running.

**Comment by patrikurban** [comment]
Hi,

I am trying to connect to your Slack but am unable to do so. Can you guide me through the steps please?

I am playing with Metarank and I cannot make it save the trained models locally. I am interacting with the .jar application on my local machine and everything goes fine if I leave both `state` and `train` as `type: memory`, but when I try `type: file`, the steps `import` and `train` work fine but `serve` gives me `java.lang.Exception: cannot load model: not found, maybe you forgot to run train?`. I was trying to get to your Slack to see, if someone else had the same problem and how they solved it...

thanks
